hawk/src/main/java/com/orhanobut/hawk/Base64Encryption.java

```diff
@@ -0,0 +1,26 @@
+package com.orhanobut.hawk;
+
+/**
+ * Provides Base64 Algorithm
+ */
+public class Base64Encryption implements Encryption {
+  @Override
```

Comment:
as convention, please put the override annotation in the same line along with method.
```java
@Override public boolean init()
```